packages/react/src/themes/teams/siteVariables.ts

```diff
@@ -25,6 +25,14 @@ export const red = colors.red[900] // no mapping color $app-red
 export const red08 = '#F3D6DB' // no mapping color $app-red-08
 export const red10 = '#FCF4F6' // no mapping color $app-red-10
 
+//
+// BORDER STYLES
+//
+export const borderWidth = '1px'
```

Comment:
Can we be more specific with the name, by replacing the `borderWidth` with `focusBorderWidth` and the `borderRadius` with `focusBorderRadius`?

Reply:
Or if these represent the values for all borders, not just focused one, then it is ok :)

Reply:
they are values for all borders, indeed, resolving